docs/reference/aggregations/metrics/top-metrics-aggregation.asciidoc

```diff
@@ -70,10 +70,21 @@ request. So,
 ==== `metrics`
 
 `metrics` selects the fields of the "top" document to return. You can request
-a single metric with something like `"metric": {"field": "m"}` or multiple
-metrics by requesting a list of metrics like `"metric": [{"field": "m"}, {"field": "i"}`.
-The fields can be <<number,numbers>>, <<keyword,keywords>>, or <<ip,ips>>.
-Here is a more complete example:
+a single metric with something like `"metrics": {"field": "m"}` or multiple
+metrics by requesting a list of metrics like `"metrics": [{"field": "m"}, {"field": "i"}`.
+
+`metrics.field` supports the following field types:
+
+* <<boolean,`boolean`>>
+* <<ip,`ip`>>
+* <<keyword,keywords>>
+* <<number,numbers>>
+
+<<runtime,Runtime fields>> of corresponding types are also supported.
+`metrics.field` doesn't support fields with <<array,array values>>. A
+`top_metric` aggregation on array values may return inconsistent results.
```

Comment:
I don't believe you can use `keyword` runtime fields. Its worth double checking, but I believe those don't work because they don't have ordinals so it can't.

Reply:
Thanks for catching this. I tested IP and integers, but not keywords. They are keyword runtime fields are not supported. I've pushed a couple of commits to address.